writers/local/local.go

```diff
@@ -22,7 +22,8 @@ type Local struct {
 	closed bool
 	config *Config
 	file   source.ParquetFile
-	writer *writer.ParquetWriter
+	writer *writer.CSVWriter
+	stream *protocol.Stream
```

Comment:
```suggestion
	stream protocol.Stream
```

pointer to an interface?